### PR TITLE
feat: add radar tracks msgs converter

### PR DESCRIPTION
**Pull request by scepter914**
## Description

Make radar_tracks_msgs_converter, new feature for radar perception.

## Tests performed

Confirm on rviz.

Screenshot from 2022-06-01 01-28-17

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].
- [ ] The PR has been properly tested.
- [ ] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: perception/radar_tracks_msgs_converter/src/radar_tracks_msgs_converter_node/radar_tracks_msgs_converter_node.cpp [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+    {
+      // Copy to local variable
+      auto p = node_param_;
+
+      // Update params
+      update_param(params, "update_rate_hz", p.update_rate_hz);
+      update_param(params, "use_twist_compensation", p.use_twist_compensation);
+
+      // Copy back to member variable
+      node_param_ = p;
```
Comment (yukkysaito) [inline perception/radar_tracks_msgs_converter/src/radar_tracks_msgs_converter_node/radar_tracks_msgs_converter_node.cpp:112]:
```suggestion
      auto & p = node_param_;

      // Update params
      update_param(params, "update_rate_hz", p.update_rate_hz);
      update_param(params, "use_twist_compensation", p.use_twist_compensation);
```

Comment (scepter914) [reply perception/radar_tracks_msgs_converter/src/radar_tracks_msgs_converter_node/radar_tracks_msgs_converter_node.cpp]: fixed

File: perception/radar_tracks_msgs_converter/src/radar_tracks_msgs_converter_node/radar_tracks_msgs_converter_node.cpp [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+    tracked_object.classification.emplace_back(classification);
+
+    tracked_objects.objects.emplace_back(tracked_object);
+  }
+  return tracked_objects;
+}
+
+uint8_t RadarTracksMsgsConverterNode::convertClassification(const uint16_t classification)
+{
+  if (classification == 32000) {
```
[the remark this answers is not in the document]
Comment (scepter914) [reply perception/radar_tracks_msgs_converter/src/radar_tracks_msgs_converter_node/radar_tracks_msgs_converter_node.cpp]: fixed

**Review by yukkysaito — APPROVED** [review]
LGTM

**Pull request merged**